internal/auth0/prompt/resource_screen_partials.go

```diff
@@ -43,7 +43,7 @@ func NewScreenPartialsResource() *schema.Resource {
 					"Options are: `" + strings.Join(allowedPromptsWithPartials, "`, `") + "`.",
 			},
 			"screen_partials": {
-				Type:     schema.TypeList,
```

Comment:
I'm not sure but did you run some tests on this?
I believe `TypeList` is ordered and `TypeSet` is a non-ordered vessel.

Ref: 
https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-types#typeset

Reply:
Reverted Back to TypeList.